lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -97,10 +97,17 @@ export class PopupClient extends StandardInteractionClient {
             const authClient: AuthorizationCodeClient = await this.createAuthCodeClient(serverTelemetryManager, validRequest.authority, validRequest.azureCloudOptions);
             this.logger.verbose("Auth code client created");
 
+            const isNativeBroker = NativeMessageHandler.isNativeAvailable(this.config, this.logger, this.nativeMessageHandler, request.authenticationScheme);
+            // Start measurement for server calls with native brokering enabled
+            let fetchNativeAccIdMeasurement;
+            if (isNativeBroker) {
+                fetchNativeAccIdMeasurement = this.performanceClient.startMeasurement(PerformanceEvents.FetchAccountIdWithNativeBroker, request.correlationId);
```

Comment:
What do you want to measure here? Not, we currently aren't capturing top level measurements for popup flows, do we want to do that?

I would prefer that you start a measurement at the beginning of this function, and then set the `isNativeBroker` property when it ends.

Reply:
Yes, we want to start for popup. I was trying to be least invasive, but I can start a top level measurement as needed. What we are measuring is a webflow to server with native brokering on, which returns us an account id instead of code.

Reply:
Done. Added toplevel and this measurement.